packages/@cdktf/hcl2cdk/lib/__tests__/expressions.test.ts

```diff
@@ -22,6 +23,48 @@ const nodeIds = [
   "local.service_name",
 ];
 
+// Transforms a path with segments into literals describing the path
```

Comment:
Moved from expression.ts since we weren't using it anywhere else other than in this test suite